content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Add new driver support matrix
+description: "Document to add new delfin driver support matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"]
+---
+
+Find below steps to add new delfin driver support matrix
+1) Clone the [documentation repository](https://github.com/sodafoundation/documentation.git)
+2) Navigate to the folder structure content/support-matrix/delfin
+3) Create a new folder with `vendor` name. For example `dell-emc`. 
+4) Create a new file `_index.md`
+5) Add the frontmatter as follows to add new Vendor. Change the content as required.
+    ```
+    ---
+    title: Dell EMC
+    description: "Support matrix data for Dell EMC Models can be found here"
+    weight: 5
+    disableToc: false
+    ---
+    
+    Support matrix data for Dell EMC Models can be found here
+    
+    ```
+
+{{% notice note %}}
+Please ensure you copy the entire frontmatter including both the `---`
+{{% /notice %}}
+    
+{{% notice note %}}
+Step 3, 4 and 5 are not required if there is already vendor folder name exists in the documentation repository
+{{% /notice %}}
+
+6) Under vendor folder name, create a new folder with `model` name. For example `VNX`.
+7) Create a new file `_index.md`. 
+8) Add the frontmatter as follows. Change the content as required.
+    ```
+    ---
+    title: VNX
+    menuTitle: VNX 
+    description: "VNX Support Matrix"
+    weight: 20
+    disableToc: false
+    tags: ["support matrix", "delfin", "drivers", "dell emc"]
+    ---
+    ```
+    
+{{% notice note %}}
+Please ensure you copy the entire frontmatter including both the `---`
+{{% /notice %}}
+
+9) To add all the support matrix tables content, copy lines from 57 to 432 from a file content/support-matrix/delfin/add-driver-support/_index.md 
+10) Fill appropriate driver data to support matrix tables. 
+11) Please find below sample driver support matrix data tables with sample data for reference to fill specific driver support matrix data.
+
+### Model and management software versions supported
+
+<table>
+   <thead>
+      <tr>
+         <th>Models</th>
+         <th>Management Software Versions</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Sample model name</td>
+         <td>Management software version</td>
+      </tr>
+      <tr>
+         <td></td>
```

Comment:
Remove extra blank row. 
OR
**Question**: Will the Management software versions be a comma separated list?
If the blank row is for reference you can write something like:
|Model name|Management Software Versions|
|---------------------------|----------------------------------------------|
|< Enter model name here>|< Management software versions in comma separated list >|
|< Enter model name here>|< Management software versions in comma separated list >|

Reply:
Never mind @Shruthi-1MN 
I saw [this](https://deploy-preview-140--soda-docs.netlify.app/support-matrix/delfin/dell-emc/vmax/) and realised it is the other way around.

![image](https://user-images.githubusercontent.com/19162717/109105468-fe74f200-7753-11eb-887c-d538d3544df4.png)

Reply:
Please do remove the extra blank rows from the template.

Reply:
done